app/src/main/kotlin/com/proxerme/app/helper/MaterialDrawerHelper.kt

```diff
@@ -228,6 +267,10 @@ class MaterialDrawerHelper(context: Activity, toolbar: Toolbar,
 
             if (item.isSelectable) {
                 currentItem = newItem
+
+                if(miniDrawer != null && newItem in this.stickyItemIds) {
```

Comment:
Pleace hit `strg` + `alt` + `L` to auto format if using Android Studio. There were also issues with imports not being properly formatted which would be fixed automatically.